client/src/components/ItemCard/ItemCard.tsx

```diff
@@ -15,60 +21,72 @@ function ItemCard({
   const theme = useTheme();
   const themeSpacing = theme.spacing(1);
 
-  const increaseCounter = () => {
-    updateItemQuantity(cartItem.item.barcode, cartItem.quantity + 1);
-  };
-
-  const decreaseCounter = () => {
-    updateItemQuantity(cartItem.item.barcode, cartItem.quantity - 1);
+  const updateItemQuantityWrapper = (quantity: number) => {
+    updateItemQuantity(cartItem.item.barcode, quantity);
   };
 
   return (
-    <Card
+    <Paper
+      elevation={0}
       style={{
         marginTop: themeSpacing,
         padding: themeSpacing,
       }}
     >
-      <Grid container direction="row" justify="center" alignItems="center">
-        <Grid item xs={3}>
-          <p>Media</p>
-        </Grid>
-        <Grid item xs={3}>
-          <Typography variant="body1">{cartItem.item.name}</Typography>
+      <Grid container>
+        <Grid item>
+          <ItemCardMedia
+            height={ITEM_CARD_MAX_HEIGHT}
+            media={cartItem.item.media}
+          />
         </Grid>
-        <Grid item xs={2}>
-          <Typography variant="body1">
-            ${cartItem.item.price.toFixed(PRICE_FRACTION_DIGITS)}
-          </Typography>
-        </Grid>
-        <Grid item xs={2}>
-          <Typography variant="body1">${getSubtotalPrice(cartItem)}</Typography>
-        </Grid>
-        <Grid
-          container
-          item
-          xs={2}
-          spacing={0}
-          direction="column"
-          alignItems="center"
-        >
-          <Grid item xs={12}>
-            <button id="inc" onClick={increaseCounter}>
-              Plus
-            </button>
-          </Grid>
-          <Grid item xs={12}>
-            <p>{cartItem.quantity}</p>
+        <Grid item xs container direction="row">
+          <Grid item xs container direction="column">
+            <Grid item xs>
+              <Typography variant="body1">{cartItem.item.name}</Typography>
+            </Grid>
+            <Grid item xs style={{ minHeight: 50, overflow: "scroll" }}>
+              <Typography variant="subtitle2">
+                ITEM DESCRIPTION
+                <br />
+                that is very
+                <br />
+                very
+                <br />
+                long
+                <br />
```

Comment:
[WIP] I'm replacing this with a `.details`/`.info` field within Item interface. Will be updating shortly

Reply:
amended to get `item.detail` instead